lasagna-master/lasagna-master.js

```diff
@@ -1,6 +1,7 @@
 /// <reference path="./global.d.ts" />
 
-import { createUnzip } from "zlib"
+// eslint-disable-next-line no-unused-vars
```

Comment:
In this case, I think we should just delete line 4 instead of turning off the linter rule.